setup.cfg

```diff
@@ -25,7 +25,7 @@ zip_safe = False
 packages = find:
 install_requires=
     gitpython>3
-    dulwich>=0.20.38
+    dulwich<=0.20.39
```

Comment:
let's add a note that dulwich build process is breaking from time to time and so to ensure that we will always have wheels, we will bump this manually from time to time

Reply:
Status for `0.20.240`: https://github.com/jelmer/dulwich/actions/runs/2353092675